projects/kubernetes/e2e/Makefile

```diff
@@ -0,0 +1,29 @@
+ORG?=docker
```

Comment:
Should be `linuxkitprojects` I think.

Reply:
Or perhaps since this is not really a linuxkit "package" but rather a service container of sorts maybe it belongs somewhere else, not sure.

Reply:
Right now this image doesn't get published, not sure if it will need to be published at some point.